daml-lf/spec/daml-lf-1.rst

```diff
@@ -1109,7 +1109,7 @@ for the ``DefTemplate`` rule). ::
    Γ  ⊢  'no_key'
 
     ⊢ₛ τ      Γ  ⊢  eₖ  :  τ
-    ⊢ₖ eₖ                                                               [DAML-LF 1.3]
+    ⊢ₖ eₖ                                                               [DAML-LF < 1.4]
```

Comment:
The one before was right, wasn't it? We had those kind of CKs for exactly one version, I think.

I guess they're both fine, but I'm curious on why you changed it.

Reply:
I changed it because ``[DAML-LF 1.3]`` does not respect the convention describe in the intro.

> [...] In addition, we will
> mark lines within inference rules with annotations of the form
> ``[DAML-LF < x.y]`` and ``[DAML-LF ≥ x.y]`` to make the respective
> line conditional upon the DAML-LF version.```

Reply:
I see, that's fair.